db/import_mb_data.py

```diff
@@ -0,0 +1,1512 @@
+        "name": value[2],
+        "type": value[3],
+        "edits_pending": value[4],
+        "last_updated": value[5],
+        "begin_date_year": value[6],
+        "begin_date_month": value[7],
+        "begin_date_day": value[8],
+        "end_date_year": value[9],
+        "end_date_month": value[10],
+        "end_date_day": value[11],
+        "ended": value[12],
+        "comment": value[13]} for value in MB_area_data
+    ]
+    connection.execute(area_query, values)
+    transaction.commit()
+    print("INSERTED area data\n")
+
+
+def write_begin_area(transaction, connection):
+    begin_area_query = text("""
+        INSERT INTO musicbrainz.area
+            VALUES (:id, :gid, :name, :type, :edits_pending, :last_updated, :begin_date_year,
+                    :begin_date_month, :begin_date_day, :end_date_year, :end_date_month, :end_date_day,
+                    :ended, :comment)
+    """)
+    values = [{
+        "id": value[0],
+        "gid": value[1],
+        "name": value[2],
+        "type": value[3],
+        "edits_pending": value[4],
+        "last_updated": value[5],
+        "begin_date_year": value[6],
+        "begin_date_month": value[7],
+        "begin_date_day": value[8],
+        "end_date_year": value[9],
+        "end_date_month": value[10],
+        "end_date_day": value[11],
+        "ended": value[12],
+        "comment": value[13]} for value in MB_begin_area_data
+    ]
+    connection.execute(begin_area_query, values)
+    transaction.commit()
+    print("INSERTED begin_area data\n")
+
+
+def write_end_area(transaction, connection):
+    end_area_query = text("""
+        INSERT INTO musicbrainz.area
+            VALUES (:id, :gid, :name, :type, :edits_pending, :last_updated, :begin_date_year,
+                    :begin_date_month, :begin_date_day, :end_date_year, :end_date_month, :end_date_day,
+                    :ended, :comment)
+    """)
+    values = [{
+        "id": value[0],
+        "gid": value[1],
+        "name": value[2],
+        "type": value[3],
+        "edits_pending": value[4],
+        "last_updated": value[5],
+        "begin_date_year": value[6],
+        "begin_date_month": value[7],
+        "begin_date_day": value[8],
+        "end_date_year": value[9],
+        "end_date_month": value[10],
+        "end_date_day": value[11],
+        "ended": value[12],
+        "comment": value[13]} for value in MB_end_area_data
+    ]
+    connection.execute(end_area_query, values)
+    transaction.commit()
+    print("INSERTED end_area data\n")
+
+
+def write_artist(transaction, connection):
+    artist_query = text("""
+      INSERT INTO musicbrainz.artist
+          VALUES (:id, :gid, :name, :sort_name, :begin_date_year, :begin_date_month, :begin_date_day,
+                :end_date_year, :end_date_month, :end_date_day, :type, :area, :gender, :comment, :edits_pending,
+                :last_updated, :ended, :begin_area, :end_area)
+          ON conflict do nothing
```

Comment:
ON CONFLICT here should name the column which conflicts, and CONFLICT should be in caps.